### PR TITLE
Update references to support resources

### PR DESCRIPTION
**Pull request by deoren**
This is per a recent mailing list conversation.

closes rsyslog/rsyslog-doc#622

### CODE REVIEW

File: source/includes/footer.inc.rst [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
-   please review the contribution guidelines listed in the
-   |GitHubSourceProjectREADME|_.
+   Help with configuring/using |PRODUCT|:
 
-   If you have a question about these docs or |PRODUCT| in general, please
-   see the following resources:
+   - |RsyslogOfficialMailingList|_ - best route for general questions
+   - GitHub: |GitHubSourceProject|_ - best route for detailed questions and
+     for reporting issues that are believed to be bugs with |PRODUCT|
+   - |RsyslogStackOverflow|_ - least activity by dev team, but generally
```
Comment (rgerhards) [inline source/includes/footer.inc.rst]: I would tell that this is *experimental* -- it involves a kind of "use at your own risk, you have been told" kind of attitude IMO

**Comment by rgerhards** [comment]
In general I think this looks good, but we should wait over the weekend if there is objection.

File: source/includes/footer.inc.rst (hunk 1 above)
Comment (deoren) [reply source/includes/footer.inc.rst]: I can add further emphasis if you prefer/think it's necessary. I was thinking that "least activity by dev team" would cover that and would still hold true even if we end up providing more support for it over time.

**Comment by deoren** [comment]
@rgerhards Alright. I didn't say it in the OP for this PR (I meant to), but I went ahead and split out "support" and "contributions" vs how it was before. With everything together it felt a little crowded.

**Comment by deoren** [comment]
Some quick notes for later review (in case we do go this route):

Stack Overflow also provides a way to link to the "ask a question" form and include an initial tag:

    https://stackoverflow.com/questions/ask?tags=rsyslog

This is the link currently included in the PR (the listing of existing questions with the 'rsyslog' tag with a button off to the right to ask new questions):

    https://stackoverflow.com/questions/tagged/rsyslog

**Comment by rgerhards** [comment]
> Some quick notes for later review (in case we do go this route):
> 
> Stack Overflow also provides a way to link to the "ask a question" form and include an initial tag:
> 
> https://stackoverflow.com/questions/ask?tags=rsyslog
> This is the link currently included in the PR (the listing of existing questions with the 'rsyslog' tag with a button off to the right to ask new questions):
> 
> https://stackoverflow.com/questions/tagged/rsyslog

Just followed that and think the second form (existing questions) is great. I only begun to wonder if StackOverflow is really right - or another of there myriad of web properties ;-) How do you find which is the right one, or is it visible through all the domains :-S

File: source/includes/footer.inc.rst (hunk 1 above)
Comment (rgerhards) [reply source/includes/footer.inc.rst]: I would go for "experimental" as it currently looks pretty seldom that a dev will check, but it's not something I insist on...

**Comment by rgerhards** [comment]
@deoren I am also talking how we can finally change the rsyslog forum to read-only state. Ping me if you don't hear back within the next 2 days. And, yes, I am approaching a state with imfile where I again begin to have "a couple of minutes" "in between (test runs)" ;-)

**Comment by deoren** [comment]
> @rgerhards: I only begun to wonder if StackOverflow is really right - or another of there myriad of web properties ;-) 

I tried to craft the link to use the main stackexchange.com domain previously and it didn't work. Looking now I see an option for setting up a filter that applies to all sites. In the results I believe I see some rsyslog tagged questions from serverfault.com that do not show up in the list for stackoverflow.com.

> @rgerhards: How do you find which is the right one, or is it visible through all the domains :-S

I'll double-check to make sure that is true. If it is, then you're right: the link I proposed in the PR won't do quite what we're looking for.

After testing, I found that you can login and create a "filter" which appears to be usable for others without logging in. Optionally you can use the same filter to receive email notifications.

stackexchange_setup_filter_for_all_sites_2018-03-13

**Comment by deoren** [comment]
> @deoren: After testing, I found that you can login and create a "filter" which appears to be usable for others without logging in. 

If that is the route we go, I suggest that you be the one to create it either under your account or a "service" account tied back to the rsyslog.com domain. Once you specify the parameters (e.g., as shown in the previous screenshot), we can use the new link in the docs.

**Comment by deoren** [comment]
> @rgerhards: @deoren I am also talking how we can finally change the rsyslog forum to read-only state. Ping me if you don't hear back within the next 2 days. And, yes, I am approaching a state with imfile where I again begin to have "a couple of minutes" "in between (test runs)" ;-)

ACK.

**Comment by rgerhards** [comment]
@deoren on StackExchange: I realized that it officially says it is targeted towards "developers" (thus I probably knew it best). So I am not sure if that really is the place admins look at... I don't think it is necessary to have an "over the network capability" -- it's more a kind of frustration on my side that they offer too many (IMHO) brandings. Up to the point that it's too fuzzy to be useful...

**Comment by deoren** [comment]
> @rgerhards: I would go for "experimental" as it currently looks pretty seldom that a dev will check, but it's not something I insist on...

I'll add that in when I apply the next update to the file. I assume that we'll hold off from merging this PR until that is changed and we know for sure what URL we'll use to link over to Stack Exchange/Overflow/whatever.

> @rgerhards: it's more a kind of frustration on my side that they offer too many (IMHO) brandings. Up to the point that it's too fuzzy to be useful...

I agree. At some point if you subdivide too much you fracture your community.

**Comment by deoren** [comment]
I think I missed where the forum was finally put into read-only mode. I knew it was probably coming, but I missed where it happened.

I'll review this PR tomorrow and update it to reflect the recent conversations that we've had on this. I'll ping you when it's ready for review.

**Comment by rgerhards** [comment]
Maybe I forgot to notify you. Sorry if that's the case.

**Comment by deoren** [comment]
> @rgerhards: Maybe I forgot to notify you. Sorry if that's the case.

Not a problem, I wasn't trying to blame you, just noting that I've been busy with other stuff and missed it.

**Comment by deoren** [comment]
@rgerhards If you haven't yet, see  https://github.com/rsyslog/rsyslog-doc/pull/624#issuecomment-372731827.

Did you want to create such a filter for our use here? In case I didn't do a good job explaining it there, the idea is to create a "filter" (accessible by others without logging in) to create a view across all Stack Exchange properties for the `rsyslog` tag. Otherwise, as you noted it appears we would have to create multiple links to encompass them all.

**Comment by rgerhards** [comment]
I don't necessarily think that we need an across-all-sites filter. My main concerns is that stack overflow is actually the wrong subsite, see https://stackexchange.com/sites - for StackOverflow it gives the target audience as "Q&A for professional and enthusiast programmers" while for ServerFault it gives "Q&A for system and network administrators". From that I would say we target the wrong site.

**Comment by deoren** [comment]
> @rgerhards: I don't necessarily think that we need an across-all-sites filter. My main concerns is that stack overflow is actually the wrong subsite, see https://stackexchange.com/sites - for StackOverflow it gives the target audience as "Q&A for professional and enthusiast programmers" while for ServerFault it gives "Q&A for system and network administrators". From that I would say we target the wrong site.

I get what you're saying in terms of "purpose", but if you look where users are actually posting their questions I don't think they're paying as much attention to the purpose of each site as you are (which is unfortunate). I see the most 'rsyslog' tagged posts in the filtered list coming from stackoverflow.com with a spread of other properties mixed in.

Here is a sampling:

- stackoverflow.com (seems to have the most questions with that tag)
- serverfault.com
- superuser.com
- unixstackexchange.com

I created a filter and sent it to you by email to give you an idea of the results.

I'm not saying the filter is the ideal approach, but it looks like a workable one based on where users are actually posting.

**Comment by rgerhards** [comment]
Well, given that mess (no wonder @100+ sites...), I would actually find any solution good to give a try. So go ahead with whatever you find useful - you are closer to the target user base than I am.

**Comment by deoren** [comment]
> @rgerhards: Well, given that mess (no wonder @100+ sites...), I would actually find any solution good to give a try. So go ahead with whatever you find useful - you are closer to the target user base than I am.

What do you think as of the changes in afd21ef41d70cbc3c81d06d7742dd889d6c11870?

**Comment by deoren** [comment]
@rgerhards Thanks for the feedback.

The only potential "gotcha" here is the filter I setup for viewing the 'rsyslog' tagged posts against all Stack Exchange properties. I _assume_ that the filter link will keep working if my account is closed, but I don't know that for sure. It may be worth recreating the filter under your account or an adiscon.com account set aside specifically for the purpose.

If you're alright with the risk, please merge this when ready. If you believe it's worth setting up the filter using another account (one owned by adiscon.com), let me know if there is anything I can do to help with that and I'll do so. We can then replace the existing filter link currently in the substitution definitions doc with the new link.

**Comment by rgerhards** [comment]
I think we can bear the risk 😉

**Comment by deoren** [comment]
> @rgerhards: I think we can bear the risk

:)

**Pull request merged**